.github/workflows/hosted-post-merge_version_update.yml

```diff
@@ -1,15 +1,14 @@
 name: hosted-post-merge_version_update
 on:
-  pull_request:
-    branches: [main]
-    types: [closed]
+  schedule:
+    - cron '0 23 * * 0'
+  workflow_dispatch:
```

Comment:
I confirmed people from outside of our team cannot trigger this workflow by `workflow_dispatch`. :-)